src/proxy.conf.json

```diff
@@ -1,6 +1,6 @@
 {
   "/ogloszenia/api": {
-    "target": "http://localhost:3000",
```

Comment:
This should not be changed.

Reply:
Czemu w ogóle potrzebujesz robić tę zmianę lokalnie? Wszystko powinno działać bez tego.